server/engine/src/main/java/org/eclipse/lsp/cobol/dialects/ibm/TransformTreeStage.java

```diff
@@ -253,11 +253,13 @@ private void registerProcessors(AnalysisConfig analysisConfig, ProcessingContext
 
     // Phase POST DEFINITION
     ctx.register(ProcessingPhase.POST_DEFINITION, SectionNode.class, new ImplicitVariablesProcessor());
+    ctx.register(ProcessingPhase.POST_DEFINITION, FunctionDeclaration.class, new ProgramRepositoryEnricher(symbolAccumulatorService));
 
     // Phase USAGE
     ProcessingPhase u = ProcessingPhase.USAGE;
     ctx.register(u, CodeBlockUsageNode.class, new CodeBlockUsage(symbolAccumulatorService));
     ctx.register(u, QualifiedReferenceNode.class, new QualifiedReferenceUpdateVariableUsage(symbolAccumulatorService));
+    ctx.register(u, QualifiedReferenceNode.class, new FunctionUsageReferenceEnricher(symbolAccumulatorService));
```

Comment:
The more I think about this, the more I am convinced that this should run in its own phase after "post definition" but before "usage".

Reply:
updated.
NOTE: Changes in QualifiedReferenceUpdateVariableUsage are just formatting changes.